### ISSUE TITLE
tex_math_single_backslash possibly broken in 2.12?

### ISSUE THREAD

**Issue by dzackgarza** [jgm/pandoc#7176]
After some recent updates on Arch, any markdown documents that use the \\( ... \\) notation (one backslash) for inline math have started breaking. I've posted a MWE below: it seems that it's just stripping the backslashes instead of recognizing it as math, although $....$ does still seem to work:

```
cat test.md
# Thursday, March 25

We have several variants: \( y=mx+b \) and $ax+b$.

Test test.

pandoc test.md --from=markdown+tex_math_single_backslash --to=markdown

# Thursday, March 25

We have several variants: ( y=mx+b ) and $ax+b$.

Test test.

pandoc --version
pandoc 2.12
Compiled with pandoc-types 1.22, texmath 0.12.2, skylighting 0.10.5,
citeproc 0.3.0.9, ipynb 0.1.0.1
```

**Comment by tarleb** [comment]
This has been reported in #7155 and is fixed in [pandoc 2.13](https://github.com/jgm/pandoc/releases/tag/2.13). Please use a binary available from the download page until your distro ships the latest version.

**Comment by dzackgarza** [comment]
I see, thank you! For some reason that didn't show up when I looked around. I can confirm that cloning the current repo and installing 2.13 with stack seems to fix this issue.
